main.c

```diff
@@ -67,10 +27,20 @@ void testPairToNumber(
     assert(pairNumber == expectedPairNumber);
 }
 
+void colorMapperManual()
+{
+ 	 ColorPair colorPair;
+     for(int pairNumber=1; pairNumber<26; pairNumber++)
+     {
+         colorPair = GetColorFromPairNumber(pairNumber);
+         printf(KBLU"PairNumber:"KNRM"%d----->"KRED"MajorColor:"KNRM "%s""----->"KYEL"MinorColor:"KNRM"%s\n", pairNumber,MajorColorNames[colorPair.majorColor],MinorColorNames[colorPair.minorColor]);
+     }
+}
+
 int main() {
+    colorMapperManual();
```

Comment:
This file mixes test code and production code. This makes it impossible to deliver the production code (printing of manual) from the test code.